src/utils/format-date.js

```diff
@@ -25,10 +31,11 @@ export function formatDate(d) {
     value: date,
     simpleDate: `${date.getUTCDate()} ${months[date.getUTCMonth()]}`,
     formatted: `${months[date.getUTCMonth()]} ${date.getUTCDate()}, ${date.getUTCFullYear()} ${utcTimeTwelve.join(':')} ${utcAMPM}`, // UTC time
+    formattedShort: `${shortMonths[date.getUTCMonth()]} ${date.getUTCDate()}, ${date.getUTCFullYear()} ${utcTimeTwelve.join(':')} ${utcAMPM}`, // UTC time
     formattedLocal: `${months[date.getMonth()]} ${date.getDate()}, ${date.getFullYear()} ${localTimeTwelve.join(':')} ${localAMPM} (UTC ${localOffset})`, // local time
     formattedLocalShort: `${months[date.getMonth()]} ${date.getDate()}, ${date.getFullYear()} (UTC ${localOffset})`, // local time
     full: date.toUTCString(),
-    timestamp: date.getTime()
+    timestamp: date.getTime() / 1000
```

Comment:
Roll this back, as most often, we’re receiving a timestamp from the backend that already has millis trimmed.